gunicorn.sh

```diff
@@ -0,0 +1 @@
+poetry run gunicorn -b 0.0.0.0:80 -w 2 app_run:app
```

Comment:
You've added this script, but I can't see any evidence that you're using it; is that right?